src/SwooleRequestHandlerRunner.php

```diff
@@ -185,4 +227,27 @@ public function onShutdown(SwooleHttpServer $server): void
     {
         $this->dispatcher->dispatch(new Event\ServerShutdownEvent($server));
     }
+
+    /**
+     * @param mixed $data
+     */
+    private function createTaskEventFromStandardArguments(
+        SwooleHttpServer $server,
+        int $taskId,
+        int $workerId,
+        $data
+    ): TaskEvent {
+        return new TaskEvent($server, $taskId, $workerId, $data);
+    }
+
+    private function createTaskEventFromTaskObject(SwooleHttpServer $server, object $task): TaskEvent
```

Comment:
Isn't that simply a \Swoole\Server\Task instance?

Reply:
Had to hunt quite a bit to find this class defined - it's not documented!

There's one issue I have with using it as a typehint: the class is marked final, so there's no way for me to mock it. And there's no way to instantiate it with the data necessary. As such, I'm going to leave this as-is, as it allows me to assert the structure and behavior required.